src/forms/Predictions.js

```diff
@@ -115,6 +115,7 @@ const PredictionsStep = function ({ updateClientValue, navData, client, snippets
           future       = { client.future }
           time         = { 'future' } />
         <Divider className='ui section divider hidden' />
+        <Message visible warning>This tool is in testing and these numbers might not be right. If they're not, we'd appreciate your feedback. Please <DownloadFile>download the anonymized data</DownloadFile> and email it to <a href="mailto:andrew@codeforboston.org">andrew@codeforboston.org</a> with the answers to these questions: What are the correct numbers? What questions did the form miss asking that could be affecting the situation?</Message>
```

Comment:
The previous `<Message>` was actually between the header and the table. It seems to me more likely to be read that way, and to be associated with the results, but that's just my gut feeling. I'm definitely open to discussion. For now, I'll mark this as requesting changes until someone brings up a different perspective.

Reply:
I've adjusted the instructions in the issue. Again, sorry about that.